images/chromium-headful/Dockerfile

```diff
@@ -308,4 +308,7 @@ COPY server/runtime/playwright-executor.ts /usr/local/lib/playwright-executor.ts
 
 RUN useradd -m -s /bin/bash kernel
 
+# Make policy directory writable for runtime updates
+RUN chown -R kernel:kernel /etc/chromium/policies
```

Comment:
the way unikernels work these kind of permissions changes don't persist into the rootfs, so they need to happen in wrapper.sh (see pre-existing chowns in there)